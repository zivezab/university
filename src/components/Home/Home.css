.vertical-center {
    margin: 0;
    position: absolute;
    top: 20%;
    -ms-transform: translateY(-50%);
    transform: translateY(-50%);
}

.horizontal-center {
    margin: 0;
    position: absolute;
    left: 50%;
    -ms-transform: translateX(-50%);
    transform: translateX(-50%);
}