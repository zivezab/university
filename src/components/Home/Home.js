import React, { Component } from "react";

class Home extends Component {
  render() {
    return (
      <h2>
        Hello, please proceed to <a href="/search">Search University</a>
      </h2>
    );
  }
}

export default Home;
