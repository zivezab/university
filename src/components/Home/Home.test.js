import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';

import Home from './Home';

test('matches snapshot', () => {
  const wrapper = shallow(<Home />);

  expect(toJSON(wrapper)).toMatchSnapshot();
});
