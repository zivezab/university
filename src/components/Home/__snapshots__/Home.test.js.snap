// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<h2>
  Hello, please proceed to 
  <a
    href="/search"
  >
    Search University
  </a>
</h2>
`;
